.test-env

```diff
@@ -0,0 +1,40 @@
+# Configs for python sdk docker:
+PYTHON_VERSION=${PYTHON_VERSION:-3.8}
```

Comment:
@tzaffi Optional:  I'd prefer to _avoid_ exposing the Python version for the Docker build here.  If you prefer as is, feel welcomed to resolve.

Rationale:
* Since the repo defines a specific minimum Python version, I feel local customization shouldn't be encouraged.  From my perspective, exposing the environment variable gives the appearance that configuration is freely supported.
* To me, the name `PYTHON_VERSION` implies a broader scope than the reality.  In practice, it _only_ governs the Docker build and _not_ `setup.py`.  
  * If retained, I'd feel more comfortable renaming the variable to more closely reflect its scope.   

```suggestion
```

Reply:
Makes sense. Will verify.